examples/bidirectional/proto/kv.proto

```diff
@@ -3,6 +3,7 @@
 
 syntax = "proto3";
 package proto;
+option go_package = "./proto";
```

Comment:
These are somewhat unusual. I might suggest using the "managed mode" feature of Buf's generation instead: https://buf.build/docs/generate/managed-mode. That would mean you can drop this option in all the files and buf would automatically set it to something sensible. It'll almost certainly look something like this:
```yaml
managed:
  enabled: true
  go_package_prefix:
    default: github.com/hashicorp/go-plugin/examples/bidirectional/proto # go.mod path + proto output path
 ```

Reply:
Is there a way to do this but keep the slightly odd `grpctest` package name the same? I know it's only a test package, but it is technically part of the public API so I'd like to be quite strict about avoiding breaking changes, just to make the safety of the change easy to evaluate.